### PR TITLE
MNT: Clean-up README files to be more consistent

### LINKED ISSUE #96: Do not duplicate instructor info in multiple READMEs
Instructor assignments are in the root-level README, but then I also see instructor names listed in individual chapter READMEs. This only increases maintenance burden for future workshops and the risk of all the READMEs have conflicting information.

I propose that instructors to be only listed in root-level README.

### PR DESCRIPTION
**Pull request by pllim**
Fix #96 

* Add missing README files
* Remove instructors listing because they are already listed in root-level README
* Fill in missing authors as "unknown" -- If you know who they are, please comment on this PR or open follow-up PRs if this one is already merged

### CODE REVIEW

**Comment by pllim** [comment]
> we should list everyone who committed

I am not sure if that is a realistic goal here, as some materials were originally taken from STScI workshops, etc. I applied your suggestions though. Is this good enough?

File: 10-WrapUp/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+### Wrap-Up
+
+**Author**:  Unknown
```
Comment (stargaser) [inline 10-WrapUp/README.md]: Let's put Erik Tollerud & Kelle Cruz here.

Comment (pllim) [reply 10-WrapUp/README.md]: Done. Please merge if you are happy with this. 😸

Comment (stargaser) [reply 10-WrapUp/README.md]: Looks great, thanks so much @pllim!

**Pull request merged**